src/engraving/rendering/dev/tdraw.cpp

```diff
@@ -837,6 +845,36 @@ void TDraw::draw(const Expression* item, Painter* painter)
     drawTextBase(item, painter);
 }
 
+void TDraw::draw(const Fermata* item, Painter* painter)
+{
+    TRACE_DRAW_ITEM;
+    painter->setPen(item->curColor());
+    item->drawSymbol(item->symId(), painter, PointF(-0.5 * item->width(), 0.0));
+}
+
+void TDraw::draw(const FiguredBass* item, Painter* painter)
+{
+    // if not printing, draw duration line(s)
+    if (!item->score()->printing() && item->score()->showUnprintable()) {
+        for (double len : item->lineLengths()) {
+            if (len > 0) {
+                painter->setPen(Pen(FiguredBass::engravingConfiguration()->formattingMarksColor(), 3));
+                painter->drawLine(0.0, -2, len, -2);              // -2: 2 rast. un. above digits
+            }
+        }
+    }
+
+    if (item->items().size() < 1) {                                 // if not parseable into f.b. items
+        drawTextBase(item, painter);                                // draw as standard text
+    } else {
+        for (FiguredBassItem* item : item->items()) {               // if parseable into f.b. items
```

Comment:
There's a name clash here with `item`. Will lead to a compiler warning at least.

Reply:
fixed